src/xcode/ENA/ENA/Source/Scenes/ContactDiary/AddAndEditEntry/DiaryAddAndEditEntryViewModel.swift

```diff
@@ -52,13 +52,13 @@ class DiaryAddAndEditEntryViewModel {
 		case let .add(day, type):
 			switch type {
 			case .location:
-				let result = store.addLocation(name: textInput)
+				let result = store.addLocation(name: textInput, phoneNumber: "", emailAddress: "")
```

Comment:
why no default parameters for `addLocation`?

Reply:
It will never be needed in practice, because this view model will always need to set those values when the other TextFields are implemented.